linux/keyman-config/debian/control

```diff
@@ -16,7 +18,9 @@ Homepage: https://keyman.com/
 Package: python3-keyman-config
 Architecture: all
 Depends: ${misc:Depends}, ${python3:Depends}, python3-gi, gir1.2-webkit2-4.0,
- kmflcomp (>=10.99.1), dconf-cli, ibus-keyman
+ kmflcomp (>=10.99.1), dconf-cli, ibus-keyman,
+ python3-sentry-sdk <pkg.keyman-config.focal>,
+ python3-raven <pkg.keyman-config.bionic>,
```

Comment:
Should we list any of these new dependencies in the [linux/README.md](https://github.com/keymanapp/keyman/tree/master/linux#linux-requirementssetup)?

Reply:
That's included in [linux/keyman-config/README.md](https://github.com/keymanapp/keyman/tree/master/linux/keyman-config#preparing-to-run)